virttest/utils_libvirt/libvirt_keywrap.py

```diff
@@ -36,8 +37,29 @@ def get_some_aes_key_token(self):
         """
         some_key_attribute = "protkey_aes_128"
         attr_path = os.path.join(self.sysfs, some_key_attribute)
-        error, output = cmd_status_output("cat %s" % attr_path)
+        error, output = cmd_status_output(cmd="cat %s" % attr_path,
+                                          session=self.session)
         if error:
             logging.debug("Error reading from %s: %s", attr_path, output)
             return None
         return output
+
+
+def cmd_status_output(cmd, session=None, timeout=60):
+    """
+    Function to unify usage of process and ShellSession"
+
+    :param cmd: Command to issue.
+    :param session: Guest session. If empty, command is exeucted on host.
+    """
+
+    status = None
+    stdout = None
+    if session:
+        status, stdout = session.cmd_status_output(cmd, timeout=timeout)
+    else:
+        result = process.run(cmd, shell=False, ignore_status=True,
```

Comment:
suggest ignore_status may be pulled up to cmd_status_output

Reply:
@chunfuwen I would like to keep it this way the function `cmd_status_output` is not supposed to be consumed outside of the module. Actually, it could better be prefixed by "_" to make this clearer.
Let me know, thanks.